modules/xmpp/Caps.js

```diff
@@ -102,10 +106,17 @@ export default class Caps extends Listenable {
      * @param {String} feature the name of the feature.
      * @param {boolean} submit if true - new presence with updated "c" node
      * will be sent.
+     * @param {boolean} external whether this feature was added externally to the library.
      */
-    addFeature(feature, submit = false) {
+    addFeature(feature, submit = false, external = false) {
         this.disco.addFeature(feature);
```

Comment:
So external features are added in both presence and caps?

Reply:
Yep.